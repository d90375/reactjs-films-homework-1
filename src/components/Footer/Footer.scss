.footer {
  height: 100px;
  padding: 20px 0;
  background-color: white;

  .title {
    font-size: 25px;
    letter-spacing: 2px;
    line-height: 30px;
    font-weight: bold;
    text-align: center;
    text-transform: uppercase;
    color: #0aaee4;
  }

  .copyright {
    font-size: 14px;
    letter-spacing: 1px;
    color: #525252;
    text-align: center;
  }
}
