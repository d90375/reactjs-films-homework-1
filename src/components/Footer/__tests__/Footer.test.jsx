import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Footer from '../Footer';

test('ButtonView renders correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Footer />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
