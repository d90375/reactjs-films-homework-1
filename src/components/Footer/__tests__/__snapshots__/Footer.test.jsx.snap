// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonView renders correctly 1`] = `
<footer
  className="footer"
>
  <p
    className="title"
  >
    films
  </p>
  <p
    className="copyright"
  >
    Copyright Â© 2020 FILMS. Tatsiana Krautsova
  </p>
</footer>
`;
